stories/Gauge.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-children-prop */
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, number, text, select, withKnobs } from '@storybook/addon-knobs';
+import { Gauge } from '@textkernel/oneui';
+import { CONTEXTS } from '@textkernel/oneui/constants';
+
+storiesOf('Molecules|Gauge', module)
```

Comment:
Is it a molecule or an atom? Import file and this seem to disagree. 🤔 
I think this is the reason that it's position in storybook is not standard